### PR TITLE
Mac display by device index

### PR DESCRIPTION
**Pull request by cquammen**
Refactors the code to take a LG device device index. On macos, use that device index to find the display on which the render window should be created.

### CODE REVIEW

**Comment by cquammen** [comment]
Relies on https://gitlab.kitware.com/vtk/vtk/-/merge_requests/7298 to fully work.

File: vtkLookingGlassInterface.h [hunk 1]
```diff
@@ -38,8 +38,10 @@ class VTKRENDERINGLOOKINGGLASS_EXPORT vtkLookingGlassInterface : public vtkObjec
 
   // create an OpenGLRenderWindow suitable for looking glass
   // just a convenience method to handle the OS specific subclasses
-  // in a generic manner.
-  static vtkOpenGLRenderWindow* CreateLookingGlassRenderWindow();
+  // in a generic manner. The deviceIndex argument species the
```
Comment (jcfr) [inline vtkLookingGlassInterface.h:41]: species  -> specifies

**Comment by martinken** [comment]
+1

File: vtkLookingGlassInterface.h (hunk 1 above)
Comment (cquammen) [reply vtkLookingGlassInterface.h:41]: Good catch, thanks.

**Pull request merged**